lib/phoenix_live_view.ex

```diff
@@ -740,9 +740,44 @@ defmodule Phoenix.LiveView do
   also uses `assign_new/3` to fetch the `:current_user` in its `mount/3` callback,
   as above, the assign will be fetched from the parent LiveView, once again
   avoiding additional database queries.
+
+  assign_new also provides asscess to the new assigns in the fn:
+
+      assigns = assigns
+          |> assign_new(:foo, fn -> "foo")
```

Comment:
```suggestion
  Note that `fun` also provides access to the previously assigned values:

      assigns =
          assigns
          |> assign_new(:foo, fn -> "foo")
```

Just nitpick to fix "asscess"  and format the code.

Reply:
Thanks for that, my spelling is horrible at times

Reply:
No problem 👍🏻